### PR TITLE
New french translations, 2022-02-03

### PR DESCRIPTION
**Pull request by fylip22**
New french translations, 2022-02-03

### CODE REVIEW

File: avidemux_fr.ts [hunk 1]
```diff
@@ -3615,7 +3620,7 @@ Incompatibilité d&apos;installation</translation>
     <message>
         <location line="+1"/>
         <source>_Saturation:</source>
-        <translation type="unfinished"></translation>
+        <translation>-Saturation</translation>
```
Comment (eumagga0x2a) [inline avidemux_fr.ts]: Could you please restore the accelerator (the underscore), destroyed by an apparent typo?

Comment (fylip22) [reply avidemux_fr.ts]: the correction is done ([5ca74d7](https://github.com/mean00/avidemux2_i18n/pull/20/commits/5ca74d7bcb2161f6674e06afbe8d89dd4cd3373e)

**Comment by eumagga0x2a** [comment]
Thank you.

**Comment by eumagga0x2a** [comment]
It turned out to be that I missed a whole lot of issues in this update, one of which even broke compilation. You might want to have a look at  https://github.com/mean00/avidemux2_i18n/commit/fe25d6516e940b045135a2b751f08ad3cae0fdd0 and at https://github.com/mean00/avidemux2_i18n/commit/f1ad3d6aee471f3bbe88eeb6eba8db082e1cb5f2. Using Qt Linguist for all translation work would have averted all of these errors.

**Pull request merged**